packages/babel-traverse/src/path/family.js

```diff
@@ -67,6 +67,30 @@ export function getSibling(key) {
   });
 }
 
+export function getPrevSibling() {
+  return this.getSibling(this.key - 1);
+}
+
+export function getNextSibling() {
+  return this.getSibling(this.key + 1);
+}
+
+export function forEachNextSibling(callback = Function.prototype) {
```

Comment:
I've renamed the method in favor of `getAllNextSiblings` and `getAllPrevSiblings` which returns an array. But also accepts a `callback` and a `thisArgs` parameter. Once a `callback` parameter has been provided this will behave similar to `Array.prototype.map`.